src/main/java/com/ibm/watson/developer_cloud/http/HttpMediaType.java

```diff
@@ -26,118 +26,118 @@
   /**
    * Field APPLICATION_ATOM_XML. (value is "application/atom+xml")
    */
-  public final static String APPLICATION_ATOM_XML = "application/atom+xml";
```

Comment:
HttpMediaType.java is a interface, all variables already public static final, no need explicit declare